### DISCUSSION TITLE
client.send(new TdApi.ForwardMessages doesn't work anymore

### DISCUSSION THREAD

**Discussion by yarmoiseev** [tdlight-team/tdlight-java#114] [Q&A]
Hi there! I'm working on my study project. And I'm using TDLight for forwarding messages, which are match some pattern. So, cleint.send() doesn't work anymore, although in debug mode everything seems fine. I haven't done any change before, it just stopped working.  Only thing I was doing before, I was trying to pack a .jar but nothing worked except telegram login. After that, I've returned to my code and find client.send() not working anymore. I even did a revert, but it didn't help either. Do you have any idea, why?

                if (pattern.matcher(text).matches()) {
                    long[] messageId = new long[]{
                            update.message.id
                    };
                    client.send(new TdApi.ForwardMessages(ourChatId, update.message.chatId, messageId, null,
                            false, false, false), null);
                }

**Comment by cavallium** [comment]
You should handle the response to see if there is an error
